cpp/modmesh/mesh/StaticMesh.hpp

```diff
@@ -92,7 +92,15 @@ struct CellType : NumberBase<int32_t, double>
     uint8_t nedge() const { return m_nedge; }
     uint8_t nsurface() const { return m_nsurface; }
 
-    uint8_t nface() const { return 2 == m_ndim ? nedge() : nsurface(); }
+    uint8_t nface() const
```

Comment:
`nface()` should be a fast function.  It would be great if it has not any branch (if/else), but the original ternary expression already broke it.  When fixing the behavior bug, it would be great to also fix the performance issue.  Could you please take a look?  One way to do it is to compare the generated assembly.